item-search/README.md

```diff
@@ -79,11 +83,11 @@ Implementations that support `POST` should add a second link with the same struc
 
 ## Endpoints
 
-| Endpoint  | Returns         | Description                                                                                                            |
-| --------- | --------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| `/`       | Catalog         | Landing Page and root Catalog                                                                                          |
-| `/api`    | OAFeat OpenAPI  | The OpenAPI service description. The path for this endpoint is only recommended to be `/api`, but may be another path. |
-| `/search` | Item Collection | Search endpoint                                                                                                        |
+| Endpoint  | Returns                   | Description                                                                                                            |
+| --------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `/`       | Catalog                   | Landing Page and root Catalog                                                                                          |
```

Comment:
This table gets repeated quite often. Why do we need to repeat this? This is item search and as such I'd only think we need to describe search and not repeat everything fro core again...

Reply:
IIRC, it was Chris's preference that we include all the endpoints that would exist if we implemented this conformance class, not only the ones that get added by this class. The reason was that implementers may only look at this one spec, so it clearly tells them they need to implement these 3 endpoints. They may need to go look up the precise contents of / and /api in Core, but at least they know they must implement those.